### PR TITLE
User feedbacks related to unsigned transaction

### PR DESCRIPTION
**Pull request by yenda**
- in unsigned transaction tab, show the transaction sent screen when user
signs a transaction
- in send command flow, show dialog when users want to sign later

### CODE REVIEW

File: src/status_im/ui/screens/wallet/send/events.cljs [hunk 1]
```diff
@@ -21,9 +21,9 @@
   (fn [{:keys [web3] :as params}]
     (when web3
       (.sendTransaction
-        (.-eth web3)
-        (clj->js (select-keys params [:from :to :value]))
-        #()))))
+       (.-eth web3)
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/events.cljs]: indentation was right before

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/wallet/send/subs.cljs [hunk 2]
```diff
@@ -4,6 +4,13 @@
             [status-im.utils.money :as money]
             [status-im.utils.hex :as utils.hex]))
 
+(re-frame/reg-sub :wallet.sent/on-press-got-it
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/send/subs.cljs]: Comment would be nice: not sure what `on-press-got-it` means.

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 3]
```diff
@@ -26,11 +26,16 @@
...
 
-(defn sign-later []
-  (utils/show-question
-   (i18n/label :t/sign-later-title)
-   (i18n/label :t/sign-later-text)
-   #(re-frame/dispatch [:wallet/sign-transaction true])))
+(defn sign-later
+  ([]
+   (sign-later false))
+  ([from-chat?]
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/send/views.cljs]: Is this used somewhere?

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 4]
```diff
@@ -172,7 +177,9 @@
              #(re-frame/dispatch [:wallet/cancel-signing-modal])
              #(re-frame/dispatch [:wallet/sign-transaction-modal])
              in-progress?]
-            [sign-buttons amount-error to amount sufficient-funds? #(re-frame/dispatch [:navigate-back])])
+            [sign-buttons amount-error to amount sufficient-funds? (if from-chat?
+                                                                     sign-later
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/send/views.cljs]: Confusing especially because `sign-later` also performs `(re-frame/dispatch [:navigate-back])`.

**Review by jeluard — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/wallet/send/subs.cljs (hunk 2 above)
Comment (yenda) [reply src/status_im/ui/screens/wallet/send/subs.cljs]: it's better to have a good name for it do you have a better suggestion ? Maybe on-press-got-it-button-event ?

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 3 above)
Comment (yenda) [reply src/status_im/ui/screens/wallet/send/views.cljs]: yes

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 4 above)
Comment (yenda) [reply src/status_im/ui/screens/wallet/send/views.cljs]: rename sign-later -> sign-later-popup ?

**Review by jeluard — APPROVED** [review]

**Review by flexsurfer — CHANGES_REQUESTED** [review]
please do not change PR after it has been approved, we can't `dissoc` because we need `:send-transaction`

**Comment by jeluard** [comment]
@yenda Did you address @flexsurfer remark?

**Review by flexsurfer — APPROVED** [review]

**Comment by yenda** [comment]
@jeluard yes

**Pull request merged**